<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWarehouseDialog</class>
 <widget class="QDialog" name="AddWarehouseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_address_1">
     <property name="text">
      <string>Address 1</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_country">
     <property name="text">
      <string>Country</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_address_1"/>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="lineEdit_name"/>
   </item>
   <item row="2" column="5">
    <widget class="QLineEdit" name="lineEdit_state"/>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_state">
     <property name="text">
      <string>State</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_code"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_code">
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_zip">
     <property name="text">
      <string>Zip Code</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLineEdit" name="lineEdit_country"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_city">
     <property name="text">
      <string>City</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_city"/>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="lineEdit_address_2"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_address_2">
     <property name="text">
      <string>Address 2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_zip"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_code</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_address_1</tabstop>
  <tabstop>lineEdit_address_2</tabstop>
  <tabstop>lineEdit_city</tabstop>
  <tabstop>lineEdit_state</tabstop>
  <tabstop>lineEdit_zip</tabstop>
  <tabstop>lineEdit_country</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddWarehouseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddWarehouseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
